### DISCUSSION TITLE
content-type  question

### DISCUSSION THREAD

**Discussion by XTUxiongda** [minio/minio#19540] [Q&A] [answered]
i user mc cp to upload .docx  file ， the content-type is application/vnd.openxmlformats-officedocument.wordprocessingml.document
but when i  use go sdk to upload .docx file,. i find the content-type is default application/octet-stream.   why?  
and i can do what to change it ?

**Comment by harshavardhana** [comment, marked as answer]
Pass the correct content-type when uploading via SDK. check how `mc` does it based on mime-type
